docusaurus/docs/reference/backup/10-controller.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Controller Backup
+sidebar_label: Controller
+---
+
+## Backup
+
+The best way to ensure continuity is to run multiple controllers. You can recover from the catastrophic loss of all
```

Comment:
This can't merge yet

Reply:
I'll remove references to multiple controllers.